pythonx/ncm2_webcomplete.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+import vim
+from ncm2 import Ncm2Source, getLogger
+import re
+from os.path import expanduser, expandvars
+import subprocess
+
+from os.path import dirname, abspath, join, pardir, expandvars, expanduser
+from subprocess import check_output, PIPE
+from threading import Thread, Event
+
+logger = getLogger(__name__)
+
+
+def log(msg):
+    from datetime import datetime
+    timestamp = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S,%f")
+    with open('/tmp/ncm2-webcomplete.log', 'a') as file_:
+        file_.write('%s %s\n' % (timestamp, msg))
+
+
+class Source(Ncm2Source):
+    def __init__(self, nvim):
+        super(Source, self).__init__(nvim)
+        # self.executable_look = nvim.call('executable', 'look')
+        # filedir = dirname(abspath(__file__))
+        # projectdir = abspath(join(filedir, pardir))
+        # join(projectdir, 'sh', 'webcomplete')
+        self.__script = expanduser(expandvars('~/.local/bin/bt words'))
```

Comment:
I noticed you have hardcoded your script in the NCM2 source you've implemented, which makes it unusable outside your computer.

Is there a reason why you didn't get the variable from the var configuration, as you did in the deoplete source file?